zerver/lib/test_helpers.py

```diff
@@ -98,7 +102,7 @@ def queries_captured():
 
     queries = []
 
-    def wrapper_execute(self, action, sql, params=()):
+    def wrapper_execute(self, action, sql, params=()):  # type: ignore
```

Comment:
The goal is to not use `type: ignore` except to work around mypy bugs.

So when using type: ignore, we've been adding a comment linking to the mypy bug involved, so that we can clean those up once the bugs are fixed in mypy upstream.